open_schools_platform/parent_management/families/models.py

```diff
@@ -0,0 +1,28 @@
+import uuid
+
+from django.db import models
+from django.contrib.auth.models import UserManager as BUM
```

Comment:
Why we use BUM there? It is for UserManager. There we should use another base manager. For example, models.Manager